Loop/Views/Presets/EditPresetView.swift

```diff
@@ -217,7 +217,12 @@ struct EditPresetView: View {
                     )
                 }
             }
-            .onChange(of: preset) {
+            .onChange(of: preset.scheduleStartDate, { oldValue, newValue in
+                if newValue != nil {
+                    assignRepeatDays()
+                }
+            })
+            .onChange(of: preset) { oldValue, newValue in
```

Comment:
To suppress compiler warnings for unused variables
```suggestion
            .onChange(of: preset) { _, _ in
```